apps/api/v2/src/modules/jwt/jwt.service.ts

```diff
@@ -0,0 +1,40 @@
+import { Injectable } from "@nestjs/common";
+import { JwtService as NestJwtService } from "@nestjs/jwt";
+
+@Injectable()
+export class JwtService {
+  constructor(private readonly nestJwtService: NestJwtService) {}
+
+  sign(payload: Record<any, any>) {
+    const issuedAtTime = this.getIssuedAtTime();
+
+    const token = this.nestJwtService.sign(JSON.stringify({ ...payload, iat: issuedAtTime }));
+
+    return token;
+  }
+
+  signAccessToken(payload: Record<any, any>) {
+    const issuedAtTime = this.getIssuedAtTime();
+
+    const accessToken = this.nestJwtService.sign(
+      JSON.stringify({ type: "access_token", ...payload, iat: issuedAtTime })
+    );
+
+    return accessToken;
```

Comment:
I think you can directly use `return this.sign({ type: "access_token", ...payload})`

Reply:
Great idea! Done in [implement Morgans requests](https://github.com/calcom/cal.com/pull/13016/commits/4c20f7dfcde72243a985522e923670b553af668c) + also stopped using JSON.stringify since nest's jwt service sign accepts object directly.